CoffeeChat/Controller/SocialMediaViewController.swift

```diff
@@ -180,13 +180,49 @@ class SocialMediaViewController: UIViewController {
     }
 
     @objc func nextButtonPressed() {
-        userDefaults.set(true, forKey: Constants.UserDefaults.onboardingCompletion)
-        let homeVC = HomeViewController()
-        navigationController?.pushViewController(homeVC, animated: true)
+        guard let instagramHandle = instagramTextField.text, let facebookHandle = facebookTextField.text else { return }
```

Comment:
what happens if a user only wants to supply one of the social media handles / or only has one of them?